src/frontend/components/Header/Header.js

```diff
@@ -8,24 +8,19 @@ export default function Header() {
   return (
     <div className="Header">
       <div className="navbar">
-        <div className="navbar-logo">
-          <Link to="/">
-            <img className="logo-img" src={logo} alt="logo" />
-          </Link>
-        </div>
-        <div className="navbar-join">
+        <Link to="/">
```

Comment:
I see all across some inconsistency:
We name `className="Header"` and `className="Header-bottom"` but on all the others we use lower case.
Any reason behind it?

Reply:
`className="Header"` represents the component